extension/content.css

```diff
@@ -5,9 +5,6 @@ mark {
 .sentenceClass {
 	border-radius: 3px;
 }
-.keywordClass {
-	border-radius: 3px;
-}
```

Comment:
Also decided this doesn't look good; with no border radius, it looks like a slice down the tracker, which is cool. With radius, it looked like new blocks inside of it, which I didn't like as much.